ETL/Python ETL/configs/Kaggle info.txt

```diff
@@ -0,0 +1,11 @@
+Installation
```

Comment:
if possible make this a .md file, there are more ways to format and organize text